website/docs/advanced/override_grammar/basic.md

```diff
@@ -142,18 +152,23 @@ foo={a:10,b:20}
 nested={a:10,b:{c:30,d:40}}
 ```
 **IMPORTANT** Always single-quote overrides that contains dicts in the shell.
-### Sweeper syntax
 
+### Sweeper syntax
 A choice sweep is comma separated list with two or more elements: 
 ```shell script
 key=a,b                       # Simple sweep: ChoiceSweep(a, b)
 key="a,b","c,d"               # Elements can be quoted strings, ChoiceSweep("a,b", "c,d")
 key=[a,b],[c,d]               # Elements can be real lists, ChoiceSweep([a,b], [c,d])
 key={a:10, b:20},{c:30,d:40}  # and even dictionaries: ChoiceSweep([a,b], [c,d]){a,b}}
 ```
+More sweeping options are described in the [Extended Grammar page](extended).
 
 **IMPORTANT** You may need to quote your choice sweep in the shell
 
+### Functions
+Hydra supports several functions in the command line.
+See the [Extended Grammar page](extended) for more information. 
+
```

Comment:
github won't let me comment on the no change lines so putting it here

below you have `$ echo key1={a:10,b:20} key2=${HOME} key=[b]*`

but `key=[b]*` sort of format doesn't show up anywhere else (other than the grammer file), could you add a bit explanation on what it does?

Reply:
You need to consult the shell documentation to understand it :)
```
omry@Coronadev:~/tmp$ touch key=bwhahwa
omry@Coronadev:~/tmp$ echo key=[b]*
key=bwhahwa
omry@Coronadev:~/tmp$ set +o braceexpand -o noglob +o histexpand
omry@Coronadev:~/tmp$ echo key=[b]*
key=[b]*
omry@Coronadev:~/tmp$ 
```